elixir/apps/web/lib/web/components/navigation_components.ex

```diff
@@ -85,11 +85,12 @@ defmodule Web.NavigationComponents do
     """
   end
 
-  slot :bottom, required: false
+  slot(:bottom, required: false)
```

Comment:
Your formatter is broken, not because of changes you added to `elixir/.formatter.exs`, but because Elixir LS is broken in many cases. See https://github.com/elixir-lsp/vscode-elixir-ls/issues/345#issuecomment-1676742647. When that starts to happen you should just restart the editor.

And this particular formatted change should be rolled back manually now :(.

Reply:
Sigh, yeah these snuck back in. I'm using Vim so the issue is slightly different (related to not running `mix format` from root so that the existing style is picked up)